src/CanonicalChord.hs

```diff
@@ -30,6 +30,6 @@ canonicalizeChord (Chord.Chord root mqual highNat ext sus) =
     Chord root ccqual highNat ext sus
 
 
-canonicalizeQuality :: (Maybe Quality) -> HighestNatural -> Quality
+canonicalizeQuality :: Maybe Quality -> HighestNatural -> Quality
```

Comment:
Prefix type constructors are higher precedence than `->`; no need to parenthesize.